### DISCUSSION TITLE
Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by ZioMauri57** [indomus/forum#3312] [HUB: Home Assistant]
Buonasera, qualcuno riesce ad aiutarmi?
Ho acquistato per prova, una valvola termostatica sonoff TRVZB, un coordinatore zigbee 3.0 sonoff e un relè ZBMINI sempre sonoff, ho installato e configurato il tutto e funziona perfettamente, ora sto cercando di fare una automazione che, quando la valvola si apre mi faccia chiudere il relè e quando la valvola si chiude lo apre, il microrele lo uso per comandare l'accensione della caldaia. 
Con l'integrazione ufficiale zigbee ho il cambio di stato "ilde" o "heating" e con questi gestisco il relè, solo che le opzioni di configurazione della valvola sono pochissime, mentre con l'integrazione zigbee2mqtt le opzioni di configurazione sono molte, ha già persino la schedulazione, quindi vorrei usare questa, solo che questa non mi dice quando la valvola è aperta o chiusa, andando in strumenti per sviluppatori/stati lo vedo, si chiama running_state e commuta da "idle" a "heat" ma non riesco a tirarlo fuori, qualcuno sa come fare?
Grazie

**Comment by Scialla** [comment]
detta così non si capisce... è forse un attributo dell'entità? Nel caso devi crearti un sensore template.

Posta uno screenshot degli strumenti sviluppatori da dove si veda il dato che vuoi estrarre

**Reply by ZioMauri57** [reply]
Si, è un attributo dell'entità, ho creato un sensore binario che va a leggere "hvac_action" e ricevo "heating" quando la valvola è aperta e "idle" quando la valvola è chiusa e con questi riesco a gestire l'automazione per spegnere e accendere la caldaia.
